azext_iot/central/_help.py

```diff
@@ -319,6 +320,92 @@ def _load_central_api_token_help():
     """
 
 
+def _load_central_cde_help():
+    helps[
+        "iot central app cde"
+    ] = """
+        type: group
+        short-summary: Create and Manage Continuous Data Export (CDE) for an application.
+        long-summary: Continuous Data Export enables exporting IoT Central measurements, devices, and device templates data to storage account.
+    """
+
+    helps[
+        "iot central app cde create"
+    ] = """
+        type: command
+        short-summary: Create a new continuous data export
+        examples:
+        - name: Add new continuous data export
+          text: >
+            az iot central app cde create
+            --app-id {appid}
+            --display-name {displayName}
+            --sources "devices, deviceTemplates, telemetry"
+            --enable True,
+            --ep-type {epType}
+            --ep-conn {epConnStr}
+            --entity-name {entityName}
```

Comment:
Just looking at examples here and reading the --entity-name description (from a users perspective), I am confused how the arg should be used. Maybe some actual value sample(s) instead of placeholders would help?

Reply:
Q's: What can I put in --entity-name if I am not using a storage endpoint? Does the 'entity' need to exist already?

For example, if I _am_ using storage do I have to point to an existing thing -- i.e. if I point to a blob container that doesn't exist, what happens?

Reply:
EventHub, ServiceBus Queue and ServiceBus Topic all have connection strings that identify themselves in the parent namespace with the "EntityPath" part. So that part could be parsed out and fed into the name/entity-name request property.

_If_ I understand `--entity-name` correctly, it seems `--entity-name` can be optional, and possibly renamed to `--blob-container-name` or similar since only the Storage Account connection string does not include nested resources (a target blob container to export to).

Reply:
EventHub, ServiceBus Queue and ServiceBus Topic have connection string and entity names - similar model as StorageEndpoint. Would prefer to keep the option set similar